app/sagas/products.js

```diff
@@ -0,0 +1,22 @@
+import { put, call, takeLatest } from 'redux-saga/effects';
+
+import {
+  requestProducts,
+  productsSuccess,
+  productsError,
+} from '../reducers/products';
+import { fetchAllProducts } from '../Api';
+
+export function* fetchProducts() {
+  try {
+    const response = yield call(fetchAllProducts);
+    const [, ...products] = response.data;
+    yield put(productsSuccess(products));
+  } catch (e) {
+    yield put(productsError(e));
+  }
+}
```

Comment:
We should extract code duplication for fetching API. I'll open an Issue for this.